src/selenium_driverless/types/webelement.py

```diff
@@ -632,9 +632,15 @@ async def shadow_root(self):
     # RenderedWebElement Items
     async def is_displayed(self) -> bool:
         """Whether the element is visible to a user."""
-        # Only go into this conditional for browsers that don't use the atom themselves
-        size = await self.size
-        return not (size["height"] == 0 or size["width"] == 0)
+        try:
+            # Only go into this conditional for browsers that don't use the atom themselves
+            size = await self.size
+            return not (size["height"] == 0 or size["width"] == 0)
+        except CDPError as e:
+            if e.code == -32000 and 'Could not compute box model.' in e.message:
+                return False
```

Comment:
Should raise an exception in my view.
Returning `False` just isn't something a user would expect, I suppose